alphacsc/utils/tests/test_utils.py

```diff
@@ -62,6 +62,27 @@ def test_uv_D():
     assert_allclose(abs(uv / uv_hat), 1)
 
 
+def test_flip_uv():
+    rng = check_random_state(42)
+    n_times_atoms = 21
+    n_atoms = 15
+    n_channels = 30
+
+    uv = rng.randn(n_atoms, n_channels + n_times_atoms)
+    uv = prox_uv(uv, uv_constraint='separate', n_channels=n_channels)
+    D = get_D(uv, n_channels)
+    uv_flip = flip_uv(uv, n_channels)
+    D_flip = get_D(uv_flip, n_channels)
```

Comment:
Nove `D` and `D_flip` just above the `assert` check and add a comment explaining the rational.